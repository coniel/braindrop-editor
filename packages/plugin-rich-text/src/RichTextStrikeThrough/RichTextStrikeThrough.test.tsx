import React from 'react';
import { render } from '@testing-library/react';
import StrikeThrough from './RichTextStrikeThrough';

describe('RichTextLeaf', () => {
  it('should render as expected', () => {
    expect(
      render(<StrikeThrough>Strike through text</StrikeThrough>).asFragment(),
    ).toMatchSnapshot();
  });
});
