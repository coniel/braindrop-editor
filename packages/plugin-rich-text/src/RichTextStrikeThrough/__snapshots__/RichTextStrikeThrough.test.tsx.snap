// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RichTextLeaf should render as expected 1`] = `
<DocumentFragment>
  <span
    style="text-decoration: line-through;"
  >
    Strike through text
  </span>
</DocumentFragment>
`;
